src/components/Entitlements/ExpirationBanner.tsx

```diff
@@ -29,6 +29,11 @@ export const ExpirationBanner = ({ contractType, expirationDate, shutdownDate }:
     return null;
   }
 
+  // Don't show the banner if the expiration date is more than 90 days away
```

Comment:
So I think actually we should be providing them with a prod entitlements JWT, but if we still want to add a separate cap, we should do it [here](https://github.com/cortexapps/backstage-plugin/blob/dc39081add1e16108a57b9cdd6b99cb75d13ce60/src/components/Entitlements/ExpirationUtils.ts#L26-L27). Sorry if I led you astray